File: pinyougou_parent/pinyougou_manager_web/src/main/webapp/js/service/payLogService.js
app.service('payLogService', function ($http) {
    //旧的查询方法(弃置)								   
    this.findPage = function (pageNum, pageSize) {
        return $http.get('../payLog/findPage.do?pageNum=' + pageNum + '&pageSize=' + pageSize)
    };

    //查询方法,读取列表数据绑定到表单中
    this.search = function (pageNum, pageSize, searchEntity) {
        return $http.post('../payLog/findPage.do?pageNum=' + pageNum + "&pageSize=" + pageSize, searchEntity)
    };

    //删除方法
    this.removePayLogList = function (idList) {
        return $http.get('../payLog/removePayLog.do?idList=' + idList)
    };

    //根据id查询
    this.findOne = function (id) {
        return $http.get('../payLog/findOne.do?id=' + id)
    };

    //修改品牌
    this.updatePayLog = function (payLog) {
        return $http.post('../payLog/updatePayLog.do', payLog)
    };

    //新增品牌
    this.savePayLog = function (payLog) {
        return $http.post('../payLog/savePayLog.do', payLog)
    };   
});

File: pinyougou_parent/pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/PayLogService.java
package com.pinyougou.sellergoods.service;
import java.util.List;
import com.pinyougou.pojo.TbPayLog;							  
import entity.PageResult;
import entity.Result;
/**
 * payLog服务层接口
 * @author jieweili
 *
 */
public interface PayLogService {

    /**
     * 返回全部列表(已弃置)
     * @return
     */
    List<TbPayLog> findAll();
 
    /**
     * 返回分页对象(过时)
     * @param pageNum
     * @param pageSize
     * @return
     */
	 
    PageResult findPage(Integer pageNum, Integer pageSize);

    /**
     * 添加数据
     * @param payLog
     * @return
     */
    public Result savePayLog(TbPayLog payLog);

    /**
     * 根据id查询一个对象
     * @param id
     * @return
     */
    TbPayLog getPayLogById(String id);

    /**
     * 修改对象
     * @param payLog
     * @return
     */
    Result updatePayLog(TbPayLog payLog);

    /**
     * 批量删除
     * @param idList
     * @return
     */
    Result removePayLog(String[] idList);

    /**
     * 根据条件分页查询
     * @param payLog
     * @param pageNum
     * @param pageSize
     * @return
     */
    PageResult listPayLogByExample(TbPayLog payLog, Integer pageNum, Integer pageSize);
 
}
